DragaliaAPI.Test/Integration/Dragalia/SummonTest.cs

```diff
@@ -0,0 +1,208 @@
+
+        HttpResponseMessage response = await _client.PostAsync(
+            "summon/get_summon_history",
+            content
+        );
+
+        byte[] bytes = await response.Content.ReadAsByteArrayAsync();
+        SummonGetSummonHistoryResponse data =
+            MessagePackSerializer.Deserialize<SummonGetSummonHistoryResponse>(bytes);
+        _output.WriteLine(TestUtils.MsgpackBytesToPrettyJson(bytes));
+    }
+
+    [Fact]
+    public async Task SummonGetSummonList_ReturnsAnyData()
+    {
+        // Corresponds to JSON: "{}"
+        byte[] payload = new byte[] { 0x80 };
+        HttpContent content = TestUtils.CreateMsgpackContent(payload);
+
+        HttpResponseMessage response = await _client.PostAsync("summon/get_summon_list", content);
+
+        byte[] bytes = await response.Content.ReadAsByteArrayAsync();
+        SummonGetSummonListResponse data =
+            MessagePackSerializer.Deserialize<SummonGetSummonListResponse>(bytes);
+        _output.WriteLine(TestUtils.MsgpackBytesToPrettyJson(bytes));
+    }
+
+    [Fact]
+    public async Task SummonRequest_GetSummonPointData_ReturnsAnyData()
+    {
+        HttpContent content = TestUtils.CreateMsgpackContent(
+            MessagePackSerializer.Serialize(1020203)
+        );
+
+        HttpResponseMessage response = await _client.PostAsync(
+            "summon/get_summon_point_trade",
+            content
+        );
+
+        byte[] bytes = await response.Content.ReadAsByteArrayAsync();
+        SummonGetSummonPointTradeResponse data =
+            MessagePackSerializer.Deserialize<SummonGetSummonPointTradeResponse>(bytes);
+        _output.WriteLine(TestUtils.MsgpackBytesToPrettyJson(bytes));
+        _output.WriteLine(bytes.Length.ToString());
+    }
+
+    [Fact]
+    public async Task SummonRequest_SingleSummonWyrmite_ReturnsValidResult()
+    {
+        SummonRequest request = new SummonRequest(
+            1,
+            SummonExecTypes.Single,
+            1,
+            PaymentTypes.Wyrmite,
+            new PaymentTarget(1, 1)
+        );
+        SummonRequestResponseData data = await SummonRequest(request);
+        data.result_unit_list.Count.Should().Be(1);
+        _output.WriteLine(
+            data.result_unit_list[0].entity_type == 7
+                ? ((Dragons)data.result_unit_list[0].id).ToString()
+                : ((Charas)data.result_unit_list[0].id).ToString()
+        );
+        using (IServiceScope scope = _factory.Services.CreateScope())
+        {
+            ApiContext apiContext = scope.ServiceProvider.GetRequiredService<ApiContext>();
+            int size =
+                data.result_unit_list[0].entity_type == 7
+                    ? apiContext.PlayerDragonReliability.Count()
+                        + apiContext.PlayerDragonData.Count()
+                    : apiContext.PlayerCharaData.Count();
+            _output.WriteLine(size.ToString());
+        }
+    }
```

Comment:
I don't really like these tests that just output to stdout instead of containing asserts. It might be more meaningful to test that the database has been updated with the UpdateDataList contents

Reply:
the 10x does, can probably add it to these as we fix them in general. The output is very handy to quickly see the results in the Test output